services/buildbot/master/templates/root.html

```diff
@@ -1,29 +1,78 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html>
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-15">
-<title>Welcome to the Buildbot</title>
-</head>
+{% extends 'layout.html' %}
+{% import 'forms.html' as forms %}
 
-<body>
-<h1>Welcome to the Buildbot!</h1>
+{% block content %}
 
+<h1>Welcome to the Buildbot
+{%- if title -%}
+  &nbsp;for the&nbsp;
+  {%- if title_url -%}
+    <a href="{{ title_url }}">{{ title }}</a>
+  {%- else -%}
+    {{ title }}
+  {%- endif -%}
+&nbsp;project
+{%- endif -%}
+!
+</h1>
+
+<div class="column">
+
+{% if authz.advertiseAction('view', request) %}
 <ul>
-  <li>The <a href="boxes-supported?branch=trunk&num_builds=10">Latest
+  {% set item_class=cycler('alt', '') %}
+
+  <li class="{{ item_class.next() }}">The <a href="boxes-supported?branch=trunk&num_builds=10">Latest
   Builds</a> for each supported builder are here (or
   <a href="boxes-unsupported?branch=trunk&num_builds=10"> for unsupported builders</a>).
   </li>
+  <li class="{{ item_class.next() }}">All <a href="boxes-all?branch=trunk&num_builds=10">Latest
+  Builds are here</a>.
+  </li>
+  <li class="{{ item_class.next() }}">Benchmark <a href="boxes-benchmark?branch=trunk&num_builds=10">Latest
+  Builds are here</a>.
+  </li>
 
-  <li><a href="one_line_per_build">Recent Builds</a> are summarized here, one
-  per line.</li>
+  <li class="{{ item_class.next() }}"><a href="builds/">Built Packages</a></li>
+
+  </br>
+
+  <li class="{{ item_class.next() }}">The <a href="waterfall">Waterfall Display</a> will give you a
```

Comment:
Same for this ... there is no reason to have links to a bunch of useless pages here. Instead, we should just link to those things that people will actually use.